docs/pop_format_spec.md

```diff
@@ -0,0 +1,66 @@
+# Proof of Possession (PoP) Specification
+
+## Overview
+
+The Proof of Possession (PoP) mechanism verifies that the owner of an EOTS key
+pair also controls a BABY key pair. This validation requires five essential
+attributes, which are exposed by the `PoPExport` structure.
+
+## Attributes
+
+The `PoPExport` structure is defined
+[here](https://github.com/babylonlabs-io/finality-provider/blob/cc07bcd4dc434f7095668724aad6865bffe425e0/eotsmanager/cmd/eotsd/daemon/pop.go#L36).
+It contains the following fields:
+
+- `EotsPublicKey` – The EOTS public key of the finality provider, marshaled as
+a hexadecimal string. This value represents the public component of the EOTS
+key pair used in the signing process.
+- `BabyPublicKey` – The secp256k1 public key, marshaled as a base64 string.
+This key is extracted from the BABY keyring and uniquely identifies the BABY
```

Comment:
Let's use "Babylon" instead of "BABY" (across the document). "BABY" is the ticker. Ok to use it in attributes, e.g.  `BabyAddress`